src/app-layout/__tests__/global-breadcrumbs.test.tsx

```diff
@@ -42,9 +42,9 @@ function delay() {
   return act(() => new Promise(resolve => setTimeout(resolve, 10)));
 }
 
-function renderAsync(jsx: React.ReactElement) {
+async function renderAsync(jsx: React.ReactElement) {
   render(jsx);
-  return delay();
+  await waitFor(() => expect(wrapper.findAppLayout()!.find('[data-awsui-discovered-breadcrumbs="true"]')).toBeTruthy());
```

Comment:
Main part of the fix. Instead of waiting for an arbitrary time, we wait for a clue from the component that the deduplication happened